space-framework/space-provider/kubeflex/kubeflex.go

```diff
@@ -212,3 +239,26 @@ func (k *KflexWatcher) ResultChan() <-chan clusterprovider.WatchEvent {
 
 	return k.ch
 }
+
+func generateKubeConfig(config string) (string, error) {
+	cmdMktemp := exec.Command("mktemp", "-p", "/tmp/")
```

Comment:
How is this cleaned up?  Or do we don't care since it's a pod which by definition is ephemeral?

Reply:
I think /tmp is cleaned by the OS. In next PR we can push an explicit delete after we are done with using the kubeconfig. 
Also this is temporary as we switch to a client code and won't  need a file